velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -615,6 +621,32 @@ TEST_F(E2EFilterTest, combineRowGroup) {
   EXPECT_EQ(parquetReader.numberOfRows(), 5);
 }
 
+TEST_F(E2EFilterTest, configurableWriteSchema) {
+  rowType_ = ROW({"c0"}, {INTEGER()});
+  std::vector<RowVectorPtr> batches;
+  for (int i = 0; i < 5; i++) {
+    batches.push_back(std::static_pointer_cast<RowVector>(
+        test::BatchMaker::createBatch(rowType_, 1, *leafPool_, nullptr, 0)));
+  }
+
+  auto int32 = arrow::field("int32", arrow::int32());
```

Comment:
We can simply use `auto type = ROW({"int32"}, {INTEGER()})`

Reply:
Updated.